install.sh

```diff
@@ -18,6 +18,8 @@ Install import_users.sh and authorized_key_commands.
     -s group        Specify an IAM group for users who should be given sudo privileges, or leave
                     empty to not change sudo access, or give it the value '##ALL##' to have all
                     users be given sudo rights.
+    -p program      Specify your useradd program to use
+    -u "args"       Specify arguments to use with useradd.
```

Comment:
maybe document default? "args" could be the current default to make it more clear what the value is